mmv1/templates/terraform/examples/service_attachment_reconcile_connections.tf.erb

```diff
@@ -0,0 +1,70 @@
+resource "google_compute_service_attachment" "<%= ctx[:primary_resource_id] %>" {
+  name        = "<%= ctx[:vars]['service_attachment_name'] %>"
+  region      = "us-west2"
+  description = "A service attachment configured with Terraform"
+
+  domain_names             = ["gcp.tfacc.hashicorptest.com."]
+  enable_proxy_protocol    = true
+  connection_preference    = "ACCEPT_MANUAL"
+  nat_subnets              = [google_compute_subnetwork.psc_ilb_nat.id]
+  target_service           = google_compute_forwarding_rule.psc_ilb_target_service.id
+
+  consumer_reject_lists = ["673497134629", "482878270665"]
+
+  consumer_accept_lists {
+    project_id_or_num = "658859330310"
```

Comment:
What project number was used for this?

Reply:
Hi @c2thorn, for this values I just follow the example of this test https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/examples/service_attachment_explicit_projects.tf.erb

Reply:
ahh, I see. thanks.